vsintegration/src/FSharp.Editor/Common/InternalOptionBuilder.fs

```diff
@@ -0,0 +1,13 @@
+﻿module internal Microsoft.VisualStudio.FSharp.Editor.InternalOptionBuilder
+
+type InternalOptionBuilder() =
```

Comment:
@psfinaki  Is it ok to define this here or should I define it closer to the 2 refactor providers? Or does this already exist somewhere?

Reply:
@vzarytovskii do you know if we have something similar in our existing machinery?

Reply:
@SebastianAtWork thinking about this again - we can probably keep this but overall this seems a bit like an overkill. We have the builders like `asyncMaybe` and so on and with time we realized that although they save "space" they often hinder code clarity and intentions. 

So maybe keeping a ladder of option matching is not too bad in that regard.

Reply:
Undone